website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
I don't think this helps me here. In the instance of the inline method, it is a different method of variable declaration. 

Additionally, I think it's a slightly different use case - the CSRF token is assigning XML declared within the JS to a JS var. I need a predefined XML variable as a JS variable.

My goal here is to be 100% compatible with the method of declaration that Odoo SA provides in `website_animation`, which will allow for the same animable elements defined in custom themes specifically for `website_animation` will also work with this module without modification.